Anthropic.SDK/Messaging/ChatClientHelper.cs

```diff
@@ -182,11 +185,14 @@ public static List<AIContent> ProcessResponseContent(MessageResponse response)
                 switch (content)
                 {
                     case Messaging.ThinkingContent thinkingContent:
-                        contents.Add(new Anthropic.SDK.Extensions.MEAI.ThinkingContent(thinkingContent.Thinking, thinkingContent.Signature));
+                        contents.Add(new Microsoft.Extensions.AI.TextReasoningContent(thinkingContent.Thinking) 
+                        { 
+                            AdditionalProperties = { { nameof(ThinkingContent.Signature), thinkingContent.Signature } }
+                        });
                         break;
 
                     case Messaging.RedactedThinkingContent redactedThinkingContent:
-                        contents.Add(new Anthropic.SDK.Extensions.MEAI.RedactedThinkingContent(redactedThinkingContent.Data));
+                        contents.Add(new Microsoft.Extensions.AI.TextReasoningContent(redactedThinkingContent.Data));
```

Comment:
If my previous comment is on point, I think this should remain its own separate content type, unless there's something human readable and meaningful we can include as Text.